HW03/aleksei_zhuchkov/spec/hw_03_t_01_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require_relative '../aleksei_zhuchkov_hw_03_t_01'
+
+RSpec.describe 'Homework3' do
+  let(:obj_hw3) { Homework3.new }
```

Comment:
please, read about the `described_class` in rspec

Reply:
I have done it
https://github.com/labs-ruby/homeworks-2022/pull/122/commits/bb63cb7b0f14a8e9976359cf655aceba8321c666